lib/led-matrix/include/LEDMatrix.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "hw_bridge.h"
+
+// Flags
+#define START_FLASH 1UL  // 01
+#define ENDED_FLASH 2UL  // 10
+
+class LEDMatrix {
+ private:
```

Comment:
put this `private` block under the `public` block

Reply:
Will do! I'm curious, why is this significant?

Reply:
Remember that the header file is like the interface that someone who's using your library will see. Generally you'd put comments that describe how your API functions work here. Also for this reason, it's good practice to put the public functions above the private members because the user only cares about these public functions that they will call. They don't want to need to scroll down past private members they don't care about to find the library API.